### PR TITLE
Update BDK version and the sample code

### PR DESCRIPTION
**Pull request by w0xlt**
Update BDK version and the sample code to 0.26

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -116,4 +116,4 @@ fn main() {
 }
 ```
 
-More information about each component used in the code can be found in [BDK Documentation](https://docs.rs/bdk/0.15.0/bdk/index.html).
\ No newline at end of file
+More information about each component used in the code can be found in [BDK Documentation](https://docs.rs/bdk/0.26.0/bdk/index.html).
```
Comment (Tibo-lg) [inline docs/getting-started.md]:
```suggestion
More information about each component used in the code can be found in [BDK Documentation](https://docs.rs/bdk/latest/bdk/index.html).
```

Also there are several other doc link that point to 0.15.0 (lines 47-40)

Comment (notmandatory) [reply docs/getting-started.md]: I agree best to point this link to `latest`.

**Comment by notmandatory** [comment]
I think this one is ready to go after above suggestion added and also a rebase to make it easier to merge.

File: docs/getting-started.md (hunk 1 above)
Comment (w0xlt) [reply docs/getting-started.md]: Done in https://github.com/bitcoindevkit/bitcoindevkit.org/pull/136/commits/844687b65a8c25d18a70a6f53e04d1ee890f479b

**Comment by w0xlt** [comment]
Rebased and the suggestion above was added.

**Comment by notmandatory** [comment]
Thanks, looks good, but not the the 0.27 version of bdk is out. Please update cargo examples to version "0.27" this will pick up new patch versions and since we're staying on 0.27.X most likely until 1.0.0 is ready these docs won't need to be changed again right away.

**Comment by thunderbiscuit** [comment]
I assume the links now need to be updated to point to the 0.28.0 version. @w0xlt let me know if you'd like me to apply required fixes and we'll get this in before it becomes outdated again!

**Review by thunderbiscuit — APPROVED** [review]
ACK 3b22c8d78f4152db9476faec590ab0a22856f7a7.

**Pull request merged**